docs/components/modeler/about-modeler.md

```diff
@@ -7,29 +7,29 @@ keywords: ["process mapping tool"]
+Any executable process needs a **[Business Process Model and Notation (BPMN)](./bpmn/bpmn.md) diagram** designed and configured beforehand.
 
 A BPMN diagram is used to visually outline the structure and flow of a process. As a result, the process can be more easily understood by various stakeholders.
 
 In tandem, different events and implementation details (such as the conditions within a gateway or the specifications of a service task) must be configured in the model so the workflow engine understands what must be executed once the process reaches a certain task.
 
-Camunda offers a few tools to design your diagrams and implement them:
+Camunda offers two tools to design and implement your diagrams:
 
 - [Web Modeler](./web-modeler/launch-cloud-modeler.md)
 - [Desktop Modeler](./desktop-modeler/index.md)
 
-Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally, all while integrating your local development environment.
+Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally to also design, view, and edit models, all while integrating your local development environment.
 
 :::note
-Interested in editing the underlying XML of BPMN and DMN diagrams directly? Be sure to utilize Desktop Modeler, as it features an XML editor tab. The XML editor comes with its own history (undo or redo), search and replace functionality, and XML syntax highlighting.
+Interested in editing the underlying XML of BPMN and [Decision Model and Notation (DMN)](./dmn/dmn.md) diagrams directly? Be sure to utilize Desktop Modeler, as it features an XML editor tab. The XML editor comes with its own history (undo or redo), search and replace functionality, and XML syntax highlighting.
 :::
 
 In this guide, we'll demonstrate modeling BPMN diagrams using both Web Modeler and Desktop Modeler.
 
 ## Next steps
 
-- [Modeling BPMN](/guides/automating-a-process-using-bpmn.md) - Learn how to quickly model an automated process using BPMN.
-- [Camunda Forms](/guides/utilizing-forms.md) - Allows you to easily design and configure forms. Once configured, they can be connected to a user task or start event to implement a task form in your application.
+- [Modeling BPMN](/guides/automating-a-process-using-bpmn.md) - Learn how to model an automated process in this tutorial using Web Modeler with Camunda Platform 8 SaaS.
+- [Camunda Forms](/guides/utilizing-forms.md) - Allows you to design and configure forms. Once configured, they can be connected to a user task or start event to implement a task form in your application.
```

Comment:
To substantially improve this section we could ensure each bullet point encourages the user to dive into it.

#### Good

> Learn how to ...


#### Not so good.

> X is a feature that does Y.

----

```suggestion
- [Camunda Forms](/guides/utilizing-forms.md) - Design and configure user task forms to implement human task orchestration use-cases.
```